bool isChatting
bool isPlaying
int8 play_mode
